config/default.yml

```
@@ -4135,6 +4135,12 @@ Style/Not:
   VersionAdded: '0.9'
   VersionChanged: '0.20'
 
+Style/NumberedParametersLimit:
+  Description: 'Avoid excessive numbered params in a single block.'
+  Enabled: pending
+  VersionAdded: '<<next>>'
+  Max: 2
```

In my opinion, it becomes unclear what it means when it over `_1`.
```suggestion
  Max: 1
```

---

Works for me!